### ISSUE TITLE
(Enhancement) Доступ к blockcheck из-под веб-интерфейса

### ISSUE THREAD

**Issue by Miraihi** [remittor/zapret-openwrt#7]
Здравствуйте. Сразу поблагодарю за такой удобный веб-интерфейс для zapret, радует возможность делать все правки не заходя в ssh.

Пускай я лично мог бы обойтись без такой фичи, но на мой взгляд если добавить возможность проверять сайты через `blockcheck.sh` через веб-интерфейс, то это отлично дополнит арсенал веб-оболочки.

**Comment by konstantisu** [comment]
Да, автору безусловно, гигареспект за web-интерфейс. И голова меньше пухнет от конструкций оригинального текстового конфига в редакторе :) Подбор стратегий, наверное, непросто будет сделать, чтобы "красиво и удобно"... Однако на грабли я налетел с размаху - после установки в режиме hostlist долго тупил, почему вдруг "пробивает" все сайты. Оказалось, что юзерлист User hostname entries "из коробки" пустой, в отличие от "примера" в "Google hostname entries". Потому и грабли, видимо... Насколько помню, в "оригинале" там строчка вроде baddomain.com была для заполнения пустышки...

**Comment by kimyourjohn** [comment]
@ Miraihi
 
opkg install luci-app-commands
